examples/modals/persistent.py

```diff
@@ -53,10 +67,12 @@ async def on_ready(self):
             self.add_modal(FeedbackModal())
             self.persistent_modals_added = True
 
-        print(f'Logged in as {self.user} (ID: {self.user.id})')
-        print('------')
+        print(f"Logged in as {self.user} (ID: {self.user.id})")
```

Comment:
This can be made 1 print statement with a new line?

Reply:
This is how is appears in basic_bot.py and views/persistent.py

Although it could be changed in all 3 places